tools/randovania.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv
```

Comment:
These are unrelated and should be added in their own pull request

Reply:
highly recommend using individual branches for unrelated changes rather than doing everything on `main`

Reply:
Done as #7749